modules/release-notes/ref-release-notes-new-features.adoc

```diff
@@ -189,7 +186,4 @@ With this update, the `backstage-plugin-catalog-backend-module-logs` is enabled
 [id="feature-rhidp-5218"]
 == {rhsso-brand-name} `7.6` is deprecated as an authentication provider
 
-{rhsso-brand-name} ({rhsso}) `7.6` is deprecated as an authentication provider. You can continue to use {rhsso} until the end of maintenance support. For details, see link:https://access.redhat.com/support/policy/updates/jboss_notes/#p_sso[RHSSO lifecycle dates]. As an alternative, migrate to {rhbk-brand-name} `v24`.
-
-
-
+{rhsso-brand-name} ({rhsso}) `7.6` is deprecated as an authentication provider. You can continue to use {rhsso} until the end of maintenance support. For details, see link:https://access.redhat.com/support/policy/updates/jboss_notes/#p_sso[RHSSO lifecycle dates]. As an alternative, migrate to {rhbk-brand-name} `v24`.
```

Comment:
@jmagak Should this not be included in the _Deprecated functionalities_ section?

Reply:
> @jmagak Should this not be included in the _Deprecated functionalities_ section?

Sure @Gerry-Forde. I will update that.